src/DroneSafetyApi/Services/IHeatmapFactory.cs

```diff
@@ -6,8 +6,17 @@
 
 namespace DroneSafetyApi.Services
 {
+    /// <summary>
+    /// The IHeatmapFactory interface is capable of generating new <see cref="IHeatmap"/>s
```

Comment:
Why should we use a factory as opposed to using new? Say that it makes the code more testable, by depending on dependency injection. Say that this is the Factory design pattern (or mention that this class is a factory for IHeatmap instances)